content/raw/chat/identity_verification.md

```diff
@@ -0,0 +1,89 @@
+
+## Use Cases
+
+### Ask to verify identity
+
+prequisite: user has been added as a contact.
+
+1. User selects a contact
+2. User prompts to verify identity
+3. Users writes a challenge question or request (for e.g something only the real person would know, or some action that only that person would be able to perform)
+4. User sends the request to the contact, and waits for a response
+5. System notifies the identity verification has received a response (e.g see Activity Center Spec)
+6. User selects the identity verification notification item
+7. User verifies the response and selects "Confirm Identity"
+7.1 alternative: User selects "Mark untrustworthy"
+8. System confirms to user that the identity has been verified
+
+### Respond to identity verification request
+
+1. System notifies user of an identity verification request
+2. User selects the identity verification notification item
+3. System displays contact that is sending request, the verification question & prompts user to respond to the verification question
+4. User fills in the verification question and selects "Send Answer"
+4.1 alternative: User selects "Refuse Verification"
+5. System notifies user that the answer has been sent
+
+### Remove identity verification
+
+prequisite: contact identity has already been verified
+
+1. User selects a contact
+2. User selects "Remove Identity Verified status"
+3. System asks confirmation
+4. User confirms
+
+## Functional Requirements
+
+**general:**
+
+- Any profile MAY BE marked as untrustworthy
+- A profile marked as untrustworthy MAY HAVE the untrustworthy mark removed
+- A user MUST confirm the removal of an identity verification status
+- A profile added as a contact and marked as trusthworthy CANNOT be blocked
+
+**sending identity verification requests:**
+
+- A user MUST only be able to send an identity verification request to a mutual contact
+- The challenge question MUST BE at least 1 character
+- The challenge question MUST BE at most 280 characters
+- The challenge question MUST BE composed of only printable ascii characters
+- A user MUST BE able to cancel an ongoing identity verification request
+- A user CANNOT confirm identity of an identity verification request that has not received a response
+
+**receive identity verification requests:**
+
+- For a user to receive an identity verification request it MUST BE a mutual contact
+- Identity verification requests MUST display the sending user, date, and the challenge question
+- The user MUST BE able to reply to an identity verification request
+- The reply to challenge question MUST BE at least 1 character
+- The reply to challenge question MUST BE at most 280 characters
+- The reply to challenge question MUST BE composed of only printable ascii characters
+- The user MUST BE able to refuse verification of an identity verification request
```

Comment:
@John-44 if an user refuses a verification request,  can he receive another verification request immediatly? or there would be a cool-off period before an user is able to send another verification request to the same user?

Reply:
@richard-ramos there isn't any cool-off period after an identity verification request is declined before a user is able to send another identity verification request.  This is because identity verification requests can only be sent to mutual contacts, so if somebody starts spamming a mutual contact with identity verification requests, the mutual contact  (probably after telling them to f-off and stop bothering them!) can just un-friend the person who sending them these identity verification requests.